manifests/i/IBM/OpenJRE/18.0.2.1/IBM.OpenJRE.locale.en-US.yaml

```
@@ -0,0 +1,36 @@
+# Created with YamlCreate.ps1 v2.1.3 $debug=NVS1.5-1-22621-436
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: IBM.OpenJRE
+PackageVersion: 18.0.2.1
+PackageLocale: en-US
+Publisher: International Business Machines Corporation
```

```suggestion
Publisher: Semeru
```
This must match Control Panel.